app/views/static_pages/home.html.erb

```diff
@@ -5,16 +5,16 @@
 </div>
 
 <div class="row">
-  <div class="col-xs-2 col-xs-offset-5 home-find">
+  <div class="col-xs-2 col-xs-offset-5 align-center">
```

Comment:
You may want to consider, to simplify your markup:

```html
<div class="col-xs-12" id="landing-center">
  <%= link_to "Find a PlaySpace", root_path, class: "btn btn-primary btn-find" %>
  OR
  <%= link_to "Sign Up", new_users_path, class:"btn btn-default btn-sign-log std-margin-top" %>
  <%= linkn_to "Log In", login_path, class: "btn btn-success btn-sign-log" %>
</div>
```
and then style it from `css`